include-build/roborazzi-gradle-plugin/src/main/java/io/github/takahirom/roborazzi/RoborazziPlugin.kt

```diff
@@ -157,33 +174,37 @@ class RoborazziPlugin : Plugin<Project> {
       val outputDirRelativePathFromProjectProvider = outputDir.map { project.relativePath(it) }
       val resultDirFileTree =
         project.fileTree(RoborazziReportConst.resultDirPath)
-      val resultsSummaryFile =
-        project.file(RoborazziReportConst.resultsSummaryFilePath)
-      val reportFile =
-        project.file(RoborazziReportConst.reportFilePath)
+      val resultDirFileProperty =
+        project.layout.projectDirectory.dir(
+          RoborazziReportConst.resultDirPath
+        )
+      val resultSummaryFileProperty =
+        project.layout.projectDirectory.file(RoborazziReportConst.resultsSummaryFilePath)
+      val reportFileProperty =
+        project.layout.projectDirectory.file(RoborazziReportConst.reportFilePath)
 
       val finalizeTestRoborazziTask = project.tasks.register(
-        "finalizeTestRoborazzi$variantSlug",
-        object : Action<Task> {
+        /* name = */ "finalizeTestRoborazzi$variantSlug",
+        /* configurationAction = */ object : Action<Task> {
           override fun execute(t: Task) {
             t.onlyIf {
               val doesRoborazziRun = doesRoborazziRunProvider.get()
               t.infoln("Roborazzi: roborazziTestFinalizer.onlyIf doesRoborazziRun $doesRoborazziRun")
               doesRoborazziRun
             }
             t.doLast {
-              t.infoln("Roborazzi: roborazziTestFinalizer.doLast")
-              // Copy all files from outputDir to intermediateDir
-              // so that we can use Gradle's output caching
-              t.infoln("Copy files from ${outputDir.get()} to ${intermediateDir.get()}")
-              // outputDir.get().asFileTree.forEach {
-              //   println("Copy file ${it.absolutePath} to ${intermediateDir.get()}")
-              // }
-              outputDir.get().asFile.mkdirs()
-              outputDir.get().asFile.copyRecursively(
```

Comment:
The issue of not being able to access the output directory arises because we have configured build/intermediates/roborazzi as the output directory for the test task in order to use build/outputs/roborazzi as its input directory. This configuration is essential for calculating image differences. Therefore, when Gradle loads the cache, it only loads the intermediate directory from the cache. However, we did not copy the contents of the intermediate directory to the output directory. To resolve this, we now copy the intermediate directory to the output directory whenever the test task is skipped.